rita_client/src/heartbeat/mod.rs

```diff
@@ -187,12 +195,12 @@ pub fn send_udp_heartbeat() {
             Err(e) => {
                 warn!("Failed to resolve domain and get network info! {:?}", e);
             }
-            Ok((Err(e), _)) => {
+            Ok(Err(e)) => {
                 warn!("DNS resolution failed with {:?}", e);
             }
-            Ok((_, Err(e))) => {
-                warn!("Could not get network info with {:?}", e);
-            }
+            // Ok((_, Err(e))) => {
+            //     warn!("Could not get network info with {:?}", e);
+            // }
```

Comment:
commented out code to remove, but the warn message should go on line 136 so that we can log the error.